consensus/bor/valset/validator_set_test.go

```diff
@@ -0,0 +1,102 @@
+package valset
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"gotest.tools/assert"
+)
+
+var (
```

Comment:
Could we make it as a function or somehow remove copy-pasting?

Reply:
Sure. Will do that !